docs/forms-in-wagtail.md

```diff
@@ -0,0 +1,44 @@
+# Including forms into Wagtail page context
```

Comment:
I think it's helpful to have justification, but I might word it something more along these lines:

> Wagtail provides a FormBuilder module, but it cannot be used with subclasses of Page, like our CFGOVPage. This is an alternative approach that will work with our Page subclass.

The second paragraph doesn't really seem to be informative at all.